WeatherMonitor/ConfigData/JsonConfigHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Newtonsoft.Json;
+namespace WeatherMonitor.ConfigData;
+#nullable disable
+public abstract class JsonConfigHandler
+{
+    private const string FilePath = @"C:\Users\User\Desktop\FTS\WeatherService\WeatherMonitor\WeatherMonitor\ConfigData\data.json";
+
+    public static void LoadConfiguration()
+    {
+        var jsonString = File.ReadAllText(FilePath);
+        var configurationData = JsonConvert.DeserializeObject<ConfigurationData>(jsonString);
```

Comment:
you can add option here to ignore case, just add this options:
```suggestion
        var options = new JsonSerializerOptions
        {
            PropertyNameCaseInsensitive = true
        };
        var configurationData = JsonConvert.DeserializeObject<ConfigurationData>(jsonString, options);
```

to avoid using `JsonProperty` everywhere

Reply:
Great suggestion!,
 
 Since ***ConfigurationData*** class has private constructor in my code, so I am using ***JsonConvert*** not ***JsonSerilizer*** because JsonSerilizer fails with the private constructors ,
 So I did the same but with json Settings
```
var settings = new JsonSerializerSettings
{
    ContractResolver = new DefaultContractResolver()
    {
        NamingStrategy = new CamelCaseNamingStrategy()
    }
};
var configurationData = JsonConvert.DeserializeObject<ConfigurationData>(jsonString, settings);

```